src/main/java/control/TicketService.java

```diff
@@ -1,98 +1,127 @@
-package main.java.control;
+package control;
+
+import model.Entity;
+import model.User;
+import model.Client;
+import model.Admin;
+import model.Ticket;
+import model.UserDAO;
+import model.TicketDAO;
+
+import model.enums.ConcertHall;
+import model.enums.StadiumSector;
+import model.enums.TicketType;
+import org.postgresql.shaded.com.ongres.scram.client.ScramClient;
+import view.Printable;
+
+import java.lang.reflect.Field;
+import java.sql.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.UUID;
 
-import main.java.model.Ticket;
-import main.java.model.Entity;
-import main.java.model.Admin;
-import main.java.model.Client;
-import main.java.model.Email;
-import main.java.model.PhoneNumber;
-import main.java.model.CustomArrayList;
-import main.java.model.CustomHashSet;
-import main.java.model.Iteratorable;
 
+public class TicketService extends Entity implements Printable {
 
+    public static void main(String[] args) throws ClassNotFoundException {
 
+        //Homework #8: DBs
+        String URL = "jdbc:postgresql://localhost:5432/TicketServiceDB";
+        String USER_NAME = "postgres";
+        String PASSWORD = "123";
 
-import main.java.model.enums.ConcertHall;
-import main.java.model.enums.StadiumSector;
-import main.java.view.Printable;
+        Class.forName("org.postgresql.Driver"); // WHAT IS THIS LINE FOR?
 
-import java.time.LocalDateTime;
-import java.time.Month;
+        Connection connection = null;
 
+        try {
+            connection = DriverManager.getConnection(URL, USER_NAME, PASSWORD);
```

Comment:
move all connection configuration in separate class, where:
1. 
```java
    // read values from file
    private String url = ...;
    private String user = ...;
    private String password = ...;
```
2.  into this class create `static` connection in method, for example:
```java
    public static Connection getConnection() throws SQLException {
        return DriverManager.getConnection(jdbcUrl, user, password );
    }
```
3. Get connection on DAO level (in your UserDAO or TicketDAO) in all `try-catch` block :
```java
try {
Connection conn = ConnectionConfig.getConnection();
PreparedStatement preparedStatement= conn.prepareStatement(sql)) 
...}
```
4. close resources also on DAO level in `finally` block in each method

Reply:
Fixed. Thank you for clear example how to do it in a right way